examples/features/gracefulstop/README.md

```diff
@@ -0,0 +1,34 @@
+# Graceful Stop
+
+This example demonstrates how to gracefully stop a gRPC server using
+`Server.GracefulStop()`.
+
+## How to run
```

Comment:
Let's have a section up front that describes the process for doing a graceful shutdown: Call `GracefulStop`, which blocks until all currently in flight RPCs have completed.  Ensure `Stop` is called with a timeout so that even in flight RPCs are forcefully stopped eventually.  Then this section can describe the example's behavior.

Reply:
Added in the first para and moved explanation down